### PR TITLE
Add SetVariablesNeededFixingToFalse initialization mutator

### PR DESCRIPTION
**Pull request by nilsdeppe**
## Proposed changes

Uses the action being added in #3286 to initialize the `VariablesNeededFixing` GRMHD tag

### Code review checklist

- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).
- [ ] The PR lists upgrade instructions and is labeled `bugfix` or
  `major new feature` if appropriate.

### CODE REVIEW

File: src/Evolution/Systems/GrMhd/ValenciaDivClean/AddVariablesNeededFixing.hpp [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+class not_null;
+}  // namespace gsl
+/// \endcond
+
+namespace grmhd::ValenciaDivClean {
+/*!
+ * \brief Mutator used with `Initialization::Actions::AddSimpleTags` to
+ * initialize the `VariablesNeededFixing` to `false`
+ */
+struct AddVariablesNeededFixing {
```
Comment (fmahebert) [inline src/Evolution/Systems/GrMhd/ValenciaDivClean/AddVariablesNeededFixing.hpp]: For my understanding: if this is used in initialization, why isn't it in an initialization namespace? if it is resetting a variable already in the databox, why is it named `Add...`?

Comment (nilsdeppe) [reply src/Evolution/Systems/GrMhd/ValenciaDivClean/AddVariablesNeededFixing.hpp]: - I'm not sure, I can put it in an `Initialization` namespace :) I just parked it somewhere that wasn't "obvious horrendous" :shrug:
- it "adds" this in the sense of that the `Initialization::Actions::AddSimpleTags<AddVariablesNeededFixing>` has `Actions::SetupDataBox` add the simple tag and then `AddSimpleTags` sets it to the initial value by calling the mutator. Any suggestions on names or documentation are certainly welcome :)

Comment (kidder) [reply src/Evolution/Systems/GrMhd/ValenciaDivClean/AddVariablesNeededFixing.hpp]: I would name this `SetVariablesNeededFixingToFalse` as that is what this mutator does 🤷

**Review by kidder — CHANGES_REQUESTED** [review]

File: src/Evolution/Systems/GrMhd/ValenciaDivClean/AddVariablesNeededFixing.hpp (hunk 1 above)
Comment (nilsdeppe) [reply src/Evolution/Systems/GrMhd/ValenciaDivClean/AddVariablesNeededFixing.hpp]: I'm happy to do that. @fmahebert could you give feedback on the name before I do a rename? :)

Comment (fmahebert) [reply src/Evolution/Systems/GrMhd/ValenciaDivClean/AddVariablesNeededFixing.hpp]: Yes, I agree with this suggestion

**Comment by nilsdeppe** [comment]
Okay, I've rebased and squashed in the rename since that touched like 30% of the lines :P Thanks for the reviews :)

**Review by kidder — APPROVED** [review]

**Review by fmahebert — APPROVED** [review]
@kidder do you want to update the PR name to reflect the new name of the mutator?

**Pull request merged**